sugoi-api-file-store-provider/src/main/java/fr/insee/sugoi/store/file/configuration/ConstantConfiguration.java

```diff
@@ -0,0 +1,20 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package fr.insee.sugoi.store.file.configuration;
+
+public class ConstantConfiguration {
+  public static final String USER_SOURCE = "user_source";
+  public static final String APPLI_SOURCE = "application_source";
+  public static final String ORGANIZATION_SOURCE = "organization_source";
```

Comment:
Why not use the ones defined in  `sugoi-api-core/src/main/java/fr/insee/sugoi/core/configuration/ConstantConfig.java` ?